### PR TITLE
docs: sync exported markdown to RDMD site

### PR DESCRIPTION
**Pull request by rafegoldberg**
💬 [Conversation][ticket]
:---:

### 🧰  Changes

- [ ] [Bump the public RDMD site to v2](https://dash.readme.com/project/rdmd/v2/versions).
- [x] [Link to the RDMD docs](https://rdmd.readme.io/) in the demo app.
- [x] Update PR app pipeline to use "predictable" URLs.
- [x] Install `rdme` and pull in [exported Guides'](https://dash.readme.com/project/rdmd/v2/docs/getting-started) markdown.
- [x] Add a `sync` step to the release process.
- [x] Point dev server to markdown files in `docs/`.

[demo]: #demo-app-link-to-come
[ticket]: https://readmeio.slack.com/archives/C5B798RFZ/p1608328693124900?thread_ts=1608323185.116200&cid=C5B798RFZ

### CODE REVIEW

**Comment by kellyjosephprice** [comment]
Cool!

File: docs/code-blocks.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+An extended syntax for displaying multiple code blocks in a tabbed interface. These are written nearly identically to a series of vanilla markdown code snippets, except for their distinct *lack* of an additional line break separating each subsequent block:
+
+    ```javascript I'm A tab...
+    console.log('Code Tab A');
+    ```
+    ```javascript And I'm B tab!
+    console.log('Code Tab B');
+    ```
+
+<!--
```
Comment (kellyjosephprice) [inline docs/code-blocks.md:20]: Why is this being commented?

File: package.json [hunk 2]
```diff
@@ -23,7 +23,8 @@
     "test.browser": "jest --runInBand --testPathPattern '__tests__/browser'",
     "watch": "webpack -w --progress --mode production",
     "release": "npx semantic-release",
-    "release.dry": "npx semantic-release --dry-run"
+    "release.dry": "npx semantic-release --dry-run",
+    "sync": "rdme docs ./docs --version=2"
```
Comment (kellyjosephprice) [inline package.json:27]: I didn't realize we made it so simple. That's cool.

**Review by kellyjosephprice — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (rafegoldberg) [reply package.json:27]: Yeah, our [CLI tools](https://github.com/readmeio/rdme#usage) are dope af.

File: docs/code-block-tests.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
+---
+title: "Code Block Tests"
```
Comment (domharrington) [inline docs/code-block-tests.md:2]: Dont they have to have a category in here? Or not?

**Review by domharrington — APPROVED** [review]
LGTM if you and @kellyjosephprice are happy with the contents.

File: docs/code-block-tests.md (hunk 3 above)
Comment (rafegoldberg) [reply docs/code-block-tests.md:2]: Apparently they do not? Who knew! 🤷‍♂️

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.25.0-next.2</b>](/readmeio/markdown/releases/tag/v6.25.0-next.2)
<br>

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.25.0</b>](/readmeio/markdown/releases/tag/v6.25.0)
<br>

**Pull request merged**